### DISCUSSION TITLE
Yara for all , all for Yara !!

### DISCUSSION THREAD

**Discussion by Barnoux** [VirusTotal/yara#1872] [Q&A] [answered]
Hello,

Is there a limit of number of rules that we can use in a single *.yar file ?

Actually, i have a big yara file with a lot of rules. and it feels like the rule that i want to test is drowned in all the rules and the rule test is not trigeering.

![image](https://user-images.githubusercontent.com/47791676/216293638-39c1b1ea-d148-455c-beed-003af869d706.png)

this is how many rules i have: 
![image](https://user-images.githubusercontent.com/47791676/216295856-a4096447-ff9a-421b-81f6-46ecc86d563e.png)

I can't use multiples yara files since the solution i used can do that lol.

It also support only yara 3.4 => and they are not planning to upgrade it lol

**Comment by plusvic** [comment, marked as answer]
There's no limit in the number of rules you can have in a file. But how many rules are we talking about?

**Reply by Barnoux** [reply]
We are talking about 32567 rules

**Reply by plusvic** [reply]
I don't think that should be an issue.
